cortex-cms/cortex
# Factories + Spec Config + Example Spec

**Purpose**
Sets up Cortex with FactoryBot and a more fleshed-out Spec config, along with initial Factories and an example spec.

**JIRA**
N/A

**Changes**
* Improvements and fixes
  * Example `#gravatar` spec
  * Sets up FactoryBot for use in Engine
  * Initial set of Factories
  * Spec config cleanup
  * Remove/upgrade spec-related gems
  * Fix deprecation warnings
  * Initial spec files

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * N/A

* Migrations or Steps to Take on Production
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
N/A

* After
N/A

**Feature Server**
N/A

**How to Verify These Changes**
* Specific pages to visit
  * N/A

* Steps to take
  * Ensure specs run and pass

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies**
  * N/A

**Additional Information**
N/A
